multimedia/fadecandyserver/Makefile

```diff
@@ -0,0 +1,54 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fadecandyserver
+PKG_VERSION:=2.1.05
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/bluewavenet/fadecandyserver/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_HASH:= #shasum -a 256 of tar.gz of source files goes here
+PKG_HASH:= 4033711b2b21ade83013828b383ac386716b9f724e6607cb2ef4cb4b76787c71
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Rob White <rob@blue-wave.net>
+PKG_BUILD_PARALLEL:=1
+PKG_LICENSE:=MIT
+
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/fadecandyserver
+	SECTION:=multimedia
+	CATEGORY:=multimedia
+	TITLE:=FadeCandy Server
+	URL:=https://github.com/scanlime/fadecandy
+	DEPENDS:=+libpthread +librt +uclibcxx +libstdcpp +libc 
```

Comment:
Absolutely not. You can't link between libstdcpp and uclibcxx. Add
```include $(INCLUDE_DIR)/uclibc++.mk```
before package.mk and replace both with ```$(CXX_DEPENDS)```

Remove libc. This is already implicit.